src/parser/ast/ClassBodyNode.h

```diff
@@ -0,0 +1,118 @@
+    {
+        size_t objIndex = context->getRegister();
+        codeBlock->pushCode(CreateObject(ByteCodeLOC(m_loc.index), objIndex), context, this);
+
+        for (unsigned i = 0; i < m_elementList.size(); i++) {
+            ClassElementNode* p = m_elementList[i].get();
+
+            size_t destIndex = p->isStatic() ? classIndex : objIndex;
+
+            AtomicString propertyAtomicName;
+            bool hasKey = false;
+            size_t propertyIndex = SIZE_MAX;
+            if (p->key()->isIdentifier() && !p->isComputed()) {
+                if (p->kind() == ClassElementNode::Kind::Method) {
+                    propertyAtomicName = p->key()->asIdentifier()->name();
+                    hasKey = true;
+                } else {
+                    propertyIndex = context->getRegister();
+                    codeBlock->pushCode(LoadLiteral(ByteCodeLOC(m_loc.index), propertyIndex, Value(p->key()->asIdentifier()->name().string())), context, this);
+                }
+            } else {
+                propertyIndex = p->key()->getRegister(codeBlock, context);
+                p->key()->generateExpressionByteCode(codeBlock, context, propertyIndex);
+            }
+
+            size_t valueIndex = p->value()->getRegister(codeBlock, context);
+            p->value()->generateExpressionByteCode(codeBlock, context, valueIndex);
+
+            if (p->kind() == ClassElementNode::Kind::Method) {
+                if (hasKey) {
+                    codeBlock->pushCode(ObjectDefineOwnPropertyWithNameOperation(ByteCodeLOC(m_loc.index), destIndex, propertyAtomicName, valueIndex), context, this);
+                } else {
+                    codeBlock->pushCode(ObjectDefineOwnPropertyOperation(ByteCodeLOC(m_loc.index), destIndex, propertyIndex, valueIndex), context, this);
+                    context->giveUpRegister(); // for drop property index
+                }
+            } else if (p->kind() == ClassElementNode::Kind::Get) {
+                codeBlock->pushCode(ObjectDefineGetter(ByteCodeLOC(m_loc.index), destIndex, propertyIndex, valueIndex), context, this);
+                context->giveUpRegister(); // for drop property index
+            } else {
+                ASSERT(p->kind() == ClassElementNode::Kind::Set);
+                codeBlock->pushCode(ObjectDefineSetter(ByteCodeLOC(m_loc.index), destIndex, propertyIndex, valueIndex), context, this);
+                context->giveUpRegister(); // for drop property index
+            }
+
+            context->giveUpRegister(); // for drop value index
+        }
+
+        SetObjectInlineCache* inlineCache = new SetObjectInlineCache();
+        codeBlock->m_literalData.pushBack(inlineCache);
+        codeBlock->pushCode(SetObjectPreComputedCase(ByteCodeLOC(m_loc.index), classIndex, PropertyName(ctx->staticStrings().prototype), objIndex, inlineCache), context, this);
```

Comment:
@rerobika Oops.. I found `Context` pointer here similar to [default parameter](https://github.com/pando-project/escargot/pull/198#discussion_r275234353). Would you please fix it in another patch?